File: GGW/Strategy/Calc.cs
using Strategy.Interfaces;

namespace Strategy
{
    public class Calc
    {
        public IMethod Method;
        public Calc(IMethod method)
        {
            Method = method;
        }
        public void Calculate(int A, int B)
        {
            Method.Action(A, B);
        }
    }
}

File: Strategy/src/main/java/org/example/Calc.java
package org.example;

import org.example.interfaces.IMethod;

public class Calc {
    public IMethod _method;
    public Calc(IMethod method){
        _method=method;
    }
    public void Calculation(int A, int B){
        _method.Action(A,B);
    }
}
